runtime/src/chpl-cache.c

```diff
@@ -2525,7 +2534,7 @@ void cache_put(struct rdcache_s* cache,
 
   // And don't do anything if it's a zero-length
   if( size == 0 ) {
-    return;
+    return 1;
```

Comment:
We shouldn't count a zero-length transfer as either a hit or a miss.  (The comm layers don't count zero-length GETs or PUTs in the comm diags.)